Source/Core/CircleGeometry.js

```diff
@@ -57,8 +57,8 @@ define([
         if (!defined(radius)) {
             throw new DeveloperError('radius is required.');
         }
-        if (radius <= 0.0) {
-            throw new DeveloperError('radius must be greater than zero.');
+        if (radius < 0.0) {
+            throw new DeveloperError('radius must not be negative.');
         }
         //>>includeEnd('debug');
 
```

Comment:
If the circle has a radius <= 0, don't bother creating the `new EllipseGeometry` below.  And in `CircleGeometry.createGeometry`, return if `circleGeometry._ellipseGeometry` is undefined